src/index.html

```diff
@@ -13,6 +13,78 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section aria-label="Star holder">
+      <section class="stars stars--0">
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+      </section>
```

Comment:
It's recommended to separate blocks of HTML with empty lines. It makes the code easier to read. However, you shouldn't add empty lines between parent and child elements, like you did here between the `section` and `div` tags.